packages/@contentlayer/cli/src/commands/_BaseCommand.ts

```diff
@@ -22,24 +22,28 @@ export abstract class BaseCommand extends Command {
   })
 
   async execute() {
-    try {
-      if (this.clearCache) {
-        await fs.rm(core.ArtifactsDir.getDirPath({ cwd: process.cwd() }), { recursive: true })
-        console.log('Cache cleared successfully')
-      }
```

Comment:
While I was editing that file, I thought I would also wrap that function in a `tryPromise` and include it in the `pipe`